roles/custom/matrix-bridge-hookshot/tasks/main.yml

```diff
@@ -9,6 +9,12 @@
     - when: matrix_hookshot_enabled | bool
       ansible.builtin.include_tasks: "{{ role_path }}/tasks/inject_into_nginx_proxy.yml"
 
+- tags:
+    - reset-hookshot-encryption
+  block:
+    - when: matrix_hookshot_enabled | bool
+      ansible.builtin.include_tasks: "{{ role_path }}/tasks/reset_encryption.yml"
```

Comment:
As of right now, your file is named `reset_encryptioon.yml` (double `o`), while you're including `reset_encryption.yml` here.

The file needs to be renamed.

Reply:
thanks